src/expandProperty.js

```diff
@@ -15,6 +43,7 @@ function parseBorder(value, resolve) {
     } else if (
       val.match(BORDER_WIDTH) !== null ||
       val.match(LENGTH_UNIT) !== null ||
+      val.match(CALC) !== null ||
```

Comment:
This should make it possible to use `calc(..)` for the width value in the shorthand, which was not the case before due to the broken regex.